ocfweb/docs/markdown_based.py

```diff
@@ -56,5 +64,6 @@ def get_markdown_docs():
         yield Document(
             name='/' + name,
             title=meta['title'],
-            render=partial(render_markdown_doc, meta, text)
+            render=partial(render_markdown_doc, meta, text),
+            path=path,
```

Comment:
Yeah, I found that out pretty quickly, it's not as bad as it first appears, but there is a lot going on behind the scenes...